olm-catalog/serverless-operator-index/v4.19/catalog/serverless-operator/catalog.yaml

```diff
@@ -405,71 +405,12 @@ entries:
 - name: serverless-operator.v1.34.0
   replaces: serverless-operator.v1.33.0
   skipRange: '>=1.33.0 <1.34.0'
-- name: serverless-operator.v1.35.0
-  replaces: serverless-operator.v1.34.0
-  skipRange: '>=1.34.0 <1.35.0'
-- name: serverless-operator.v1.35.1
+- name: serverless-operator.v1.36.0
   replaces: serverless-operator.v1.35.0
-  skipRange: '>=1.35.0 <1.35.1'
```

Comment:
Looks like we need to manually update the file until we have stage/prod push on ocp 4.19